### ISSUE TITLE
Cards show shortly then all disappear but one (1.8.2)

### ISSUE THREAD

**Issue by hmmbob** [thomasloven/lovelace-auto-entities#179]
I have an overview page with multiple `auto-entities` cards (lovelace code [here](https://github.com/hmmbob/HomeAssistantConfig/blob/master/lovelace/old/9-overviews.yaml)), where upon page load I shortly see the cards after which all of them disappear but the 1st one in the lovelace code - that one remains visible. Rolling back to 1.7.0 fixes the issue.

Console shows 3 times same error:
```
auto-entities.js:1 Uncaught (in promise) TypeError: Cannot read property 'attributes' of undefined
    at Object.name (auto-entities.js:1)
    at auto-entities.js:1
    at Array.sort (<anonymous>)
    at HTMLElement.update_entities (auto-entities.js:194)
    at async HTMLElement.update_all (auto-entities.js:194)
```
Using Chrome 89 on Win 10, HA 2021.3.3 and auto-entities 1.8.2, but it also happens in the official HA Android app (latest beta).

Screenshot with issue:
![image](https://user-images.githubusercontent.com/33529490/110984967-1b244300-836c-11eb-9d2a-d032571b3b93.png)

Screenshot with 1.7.0:
![image](https://user-images.githubusercontent.com/33529490/110985210-75250880-836c-11eb-8b90-2aea5482a4a8.png)

**Comment by hmmbob** [comment]
Inspired by [this comment](https://github.com/thomasloven/lovelace-auto-entities/issues/175#issuecomment-797151667) I tried removing all `sort:` keys and now the cards remain visible on 1.8.2.

**Comment by thomasloven** [comment]
Should be fixed in 1.8.3

**Comment by hmmbob** [comment]
Confirmed, fixed in https://github.com/thomasloven/lovelace-auto-entities/commit/35df717ee380e10567ebd40d80b3ddafe636949c
